hackathon/models.py

```diff
@@ -24,10 +24,11 @@ class Hackathon(models.Model):
     created_by = models.ForeignKey(User,
                                    on_delete=models.CASCADE,
                                    related_name="hackathons")
-    display_name = models.CharField(default="", max_length=254)
-    description = models.TextField()
-    start_date = models.DateTimeField()
-    end_date = models.DateTimeField()
+    display_name = models.CharField(default="", max_length=254, blank=False, null=True)
```

Comment:
This was just from previous experience - when setting a field on an existing model to `blank=False` and there are existing records in the DB, I've received errors in the past when running the migrations again, or it has asked me to set a value for any existing blank fields. To resolve this, I've had to set `null=True` on the relevant fields, so this is a force of habit! I've removed `null=True` from the relevant fields now and it will be visible when I push the changes to the existing PR.

Reply:
Sure. That's fine, but I think it's either `null=True` or set a default. And as per Django documentation `default=""` for Char and TextFields is the preferred solution.

Reply:
Perfect, makes sense.  I'll keep that in mind going forward 👍